Q21/StudentProbability.rb

```diff
@@ -0,0 +1,48 @@
+def first_game
```

Comment:
Maybe you can have the 5,6 and 5,5 as variables being passed, so you don't have to duplicate code

Reply:
I have added in the parameters for the conditions